toolbox/strings.py

```diff
@@ -102,6 +178,30 @@ def is_invite(string: str, *, fullmatch: bool = True) -> bool:
     return False
 
 
+def remove_markdown(content: str, formats: MarkdownFormat = MarkdownFormat.ALL) -> str:
+    """
+    Removes the markdown formatting from Discord messages.
+
+    Parameters
+    ----------
+    content : str
+        The `str` object, which needs their content cleaned from Discord's markdown formatting.
+    formats : MarkdownFormat
+        The `IntFlag` of the formatting that needs to be removed. Default is `MarkdownFormat.ALL`
+
+    Returns
+    -------
+    str
+        The cleaned string without markdown formatting.
+    """
+    for format, (regex, replace) in FORMAT_DICT.items():
+        if formats & format:
+            matches = re.findall(regex, content)
+            for match in matches:
+                content = re.sub(replace.format(match), match, content)
```

Comment:
You may be able to get away with a single re expression here -- the `|` character is an OR operator, though it can be kinda finicky, so it might not work. That would fairly drastically improve the performance of this section if it did work though.

Reply:
Premature optimization is the root of all evil.
Do you mean making a bit boi pattern instead of having a bunch?
Or do you mean changing how re.findall() or re.sub() works?

Reply:
Actually on second look, it may be better to leave it as is considering the other operations that need to take place here.